cmd/git-sync/main.go

```diff
@@ -78,6 +80,9 @@ var flCookieFile = flag.Bool("cookie-file", envBool("GIT_COOKIE_FILE", false),
 var flGitCmd = flag.String("git", envString("GIT_SYNC_GIT", "git"),
 	"the git command to run (subject to PATH search)")
 
+var flHTTPBind = flag.String("http-bind", envString("GIT_SYNC_HTTP_BIND", ""),
+	"the bind address for git-sync's HTTP endpoint")
```

Comment:
example value here, maybe some docs.

Maybe add a TODO (or better - file a bug) to make this default-on in the next major rev?